pkg/features/features.go

```diff
@@ -27,6 +28,22 @@ func registerFeature(name, envVar string, defaultValue bool) FeatureFlag {
 		name:         name,
 		envVar:       envVar,
 		defaultValue: defaultValue,
+		unchangeable: false,
+	}
+	Flags[f.Name()] = f
+	return f
+}
+
+// registerUnchangeableFeature global registers and returns a new feature flag that is always locked to the default value.
+func registerUnchangeableFeature(name, envVar string, defaultValue bool) FeatureFlag {
+	if !strings.HasPrefix(envVar, "ROX_") {
+		panic(fmt.Sprintf("invalid env var: %s, must start with ROX_", envVar))
+	}
+	f := &feature{
+		name:         name,
+		envVar:       envVar,
+		defaultValue: defaultValue,
+		unchangeable: true,
 	}
 	Flags[f.Name()] = f
 	return f
```

Comment:
Nit: both functions have the same body but differ in one flag. Could you please reduce copy-paste by reusing the logic? This should help maintain code with less effort.